### PR TITLE
Add .travis.yml

### PR DESCRIPTION
**Pull request by ghost**
This pull requests adds auto-packaging so you will no longer need it package the file manually and you can just link to the releases section in the [_`README.md`_](https://github.com/TheDragonRing/hidecoords/blob/master/README.md) file. The only setup required before this can be merged is setting up a Personal Access Token and adding it to the `GITHUB_TOKEN` variable in the build settings.

### CODE REVIEW

**Comment by ghost** [comment]
*This pull request has been tested and the outcome was superb*

**Comment by dragonwocky** [comment]
@nathfreder interesting. If this project was bigger/more frequently updated, I would probably merge this. At the moment, however, it's pointless (all the add-on does is literally just comment out 3 lines). The only updating it would every need done is every now and then copy/pasting from the vanilla pack to stop the HUD breaking. Auto-compilation would only be necessary if committing from a device that doesn't have easy one-or-two-click `.zip` management - which I wouldn't be committing from, because I would need to download/unzip the vanilla pack to get the stuff to copy/paste in.

TLDR: The only reasons I would need this are if I was committing often from a device that I would be unable to commit from.
